dropbox/dropbox.py

```diff
@@ -185,11 +189,14 @@ def __init__(self,
                           self._HOST_CONTENT: self._api_content_hostname,
                           self._HOST_NOTIFY: self._api_notify_hostname}
 
+        self._timeout = timeout
+
     def request(self,
                 route,
                 namespace,
                 request_arg,
-                request_binary):
+                request_binary,
+                timeout=None):
```

Comment:
Please document the new parameter in the docstring. Here and in `request_json_object()`.